src/data/fin.lean

```diff
@@ -1177,6 +1194,22 @@ begin
       exact lt_succ_iff.mpr (not_lt.mp h), }, },
 end
 
+/-- `pred` commutes with `succ_above`. -/
+lemma pred_succ_above_pred {a : fin (n + 2)} {b : fin (n + 1)} (ha : a ≠ 0) (hb : b ≠ 0) {hk} :
```

Comment:
```suggestion
lemma pred_succ_above_pred {a : fin (n + 2)} {b : fin (n + 1)} (ha : a ≠ 0) (hb : b ≠ 0) (hk) :
```
I don't think `hk` should be implicit, because it isn't determined by the LHS.